thorlcr/activities/hashdistrib/thhashdistribslave.cpp

```diff
@@ -531,6 +551,9 @@ class CDistributorBase : implements IHashDistributor, implements IExceptionHandl
             numFinished = 0;
             stoppedTargets = 0;
             aborted = false;
+            numLocalRows = 0;
+            numRemoteRows = 0;
+            sizeRemoteWrite = 0;
```

Comment:
If the hashdistribute was in a child query, it would mean the published stats would be whatever the current or last child query values were. I don't think we should reset these.